### PR TITLE
BDLaethg40

### PR DESCRIPTION
**Pull request by thea-m**
And the next scroll :)

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 1]
```diff
@@ -50,35 +125,135 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    <desc>There are few scribal corrections. Rarely, characters are marked for deletion with strokes above and below
+                                        or are striked through.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <desc>The last visible line contains <!--cursive--> Brillenbuchstaben.</desc>
+                                </item>
+                                
+                                <item xml:id="e4">
+                                    <desc>Remains of writing and a drawing are visible on the upper part of the verso. Only single characters are legible anymore.
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:235]: Is 'anymore' a typo?

**Review by ElianaDalSasso — APPROVED** [review]
Thank you Thea! I have just a minor doubt.

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 2]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                        <msItem xml:id="ms_i1">
+                      <locus from="1r1" to="1r121"/>
+                            <title type="complete" ref="LIT1763Legend"/>
+                            <textLang mainLang="gez"/>
+                            <note>The legend of Susǝnyos proper is framed by lengthy protective
+                            supplications.</note>
+                            <incipit xml:lang="gez" type="inscriptio"><hi rend="rubric">በስመ፡ እግዚአብሔር፡ ሕያው፡ ነባቢ፡ ወተናጋሪ፡ ጸሎቱ፡ ለቅዱስ፡ ሱስንዮስ፡ በ</hi>እንተ፡
+                            አሰስሎ፡ ደዌ፡ እምሕፃናት፡ እለ፡ ይጠብው፡ ጥበ<supplied reason="omitted">፡</supplied> እሞሙ፡ እምኵሉ፡ 
+                                ደዊያ<hi rend="rubric"><gap reason="illegible" unit="chars" quantity="1"/>፡ ዓዲ፡ ጸሐፍኩ፡ ለብእሲ<surplus>ሲ</surplus>ት፡ ሐሐዋ፡ ዘየሐዩ፡ 
+                                ደዌ</hi><gap reason="illegible"/> ወትስቅሎ፡ ላዕሌሃ፡ ወይበቁዕ፡ በረድኤተ፡ እግዚአብሔር፡ ልዑል፡ ወክቡር፡
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:49]: I would mark ወይበቁዕ፡ with `<sic/>`, the expected form should be ወይበቍዕ፡ (the scroll has indeed ወይበቁዕ)

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 3]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                ለዛቲ፡ ጸሎት፡ እመሂ፡
+                            አንስት፡ ወእመሂ፡ <sic>አንስት፡</sic> ወወተባዕት፡ ወእመሂ፡ አዕሩግ፡ ወሕፃናት፡ ወወሬዛ፡ ልሂቅ፡ ኢይበጽሕ፡ እስከ፡ ለአለም፡ ወእምድኅረዝ፡ ሶበ፡ ኮነ፡ ሰማዕተ፡ ቅዱስ፡ ሱስንዮስ፡ በስመ፡
+                                እግዚእነ፡ ኢየሱስ<supplied reason="omitted">፡</supplied> <surplus>ኢየሱስ፡</surplus> ክርስቶስ፡ ሎቱ፡ ስብሐት፡ እስከ፡ ለዓለመ፡ አለም፡ አሜን።</explicit>
+                            <explicit xml:lang="gez" type="supplication">ኦአምላከ፡ ቅዱስ፡ ሱስንዮስ፡ አድኅና፡ እምሕማመ፡ ዓይነጥላ፡ ወዓይነናስ፡ ወዓይነናስ፡ ዓይነ፡ ዘር፡ ወውላጅ፡ ዓይነጠቢብ፡ ወቡዳ፡
+                            ወቡዳ፡ ዓይነ፡ ባርያ<surplus>ያ</surplus>፡ ወሌጌዎን፡ ድድቅ፡ ወጋኔነ፡ ቀትር፡ ምች፡ ወተላኖሽ፡ ቁረኛ፡ ወተያይዦ፡ ደ፡ 
+                            ወሸተላይ፡ መገኛ፡ ወጕሥምት፡ ውጋት፡ ወቁርጥማት፡ ፍልሸት፡ ሥራይ፡ ወድግምት፡ ወቁርጸት፡ ፌራ፡ ዘደጋ፡ ንዳድ፡ ዘቆላ፡ ወእምኵሉ፡ ደዌያት፡ 
+                            ከመ፡ ኢይቅረቡ፡ እምላዕለ፡ ዓመትከ<hi rend="rubric" rendition="#partialRubric">፨</hi> <persName ref="PRS13860HawituA" role="owner"><hi rend="rubric">ወይዘሮ ሃዋ</hi></persName>
+                                <persName ref="PRS13861WalattaH" role="owner"><hi rend="rubric"><add place="below">ወለተ፡ ሕውየት</add></hi></persName></explicit>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:68]: Check ሕውየት: is it ሕይወት?

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 4]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                ወምስ<hi rend="rubric">ሌሁ፡ ፲ወ፪ አርዳኢሁ፡
+                            ወሶበ፡ ርዕዩ፡ መልክዓ፡ ብእሲት፡ ዓራጊት፡ እንተ፡ ትነብ</hi>ር፡ በማዕዶተ፡ ፈለግ፡</incipit>
+                            <explicit xml:lang="gez">አድኅና፡ እምሕማመ፡ ደም፡ ወሾተላይ፡ ለዓመተ፡ እግዚአብሔር<supplied reason="omitted">፡</supplied> 
+                                <persName ref="PRS13860HawituA"><hi rend="rubric">ሐዊቱ አሊ</hi></persName>
+                                <persName ref="PRS13861WalattaH"><hi rend="rubric"><add place="interlinear">ወለተ፡ ሕይወት፡</add></hi></persName>
+                                ሊስ፡ አፍሊስ<supplied reason="omitted">፡</supplied> መሊስ፡ ማላሊስ፡ መላልዮስ፡ በኃይለ፡ ዝንቱ፡ አስማቲከ፡ ዘአርጋዕከ፡ ኀይለ፡ በረድ፡ ወነፋስ፡ ከማሁ፡ አርግዕ፡ ደማ፡ 
+                                ወአፅገዕ፡ ፍሬ፡ ማኅፀና፡ 
+                                እስከ<supplied reason="omitted">፡</supplied> ትወልድ፡ ወልደ፡ በጊዜሃ፡ ለዓመትከ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                <persName ref="PRS13860HawituA" role="owner"><hi rend="rubric">ሐዋ</hi></persName>
+                                <persName ref="PRS13861WalattaH" role="owner"><hi rend="rubric"><add place="interlinear">ወለተ፡ ሕውየት</add></hi></persName></explicit>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:86]: Is ሕውየት: to be marked with `<sic/>` (also above)?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea!

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml (hunk 3 above)
Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethg40.xml]: both typos, thanks!

**Comment by thea-m** [comment]
Thank you, Massimo!

**Review by MassimoVilla — APPROVED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 5]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            <textLang mainLang="gez"/>
+                            <note>The legend of Susǝnyos proper is framed by lengthy protective
+                            supplications.</note>
+                            <incipit xml:lang="gez" type="inscriptio"><hi rend="rubric">በስመ፡ እግዚአብሔር፡ ሕያው፡ ነባቢ፡ ወተናጋሪ፡ ጸሎቱ፡ ለቅዱስ፡ ሱስንዮስ፡ በ</hi>እንተ፡
+                            አሰስሎ፡ ደዌ፡ እምሕፃናት፡ እለ፡ ይጠብው፡ ጥበ<supplied reason="omitted">፡</supplied> እሞሙ፡ እምኵሉ፡ 
+                                ደዊያ<hi rend="rubric"><gap reason="illegible" unit="chars" quantity="1"/>፡ ዓዲ፡ ጸሐፍኩ፡ ለብእሲ<surplus>ሲ</surplus>ት፡ ሐሐዋ፡ ዘየሐዩ፡ 
+                                ደዌ</hi><gap reason="illegible"/> ወትስቅሎ፡ ላዕሌሃ፡ <sic>ወይበቁዕ፡</sic> በረድኤተ፡ እግዚአብሔር፡ ልዑል፡ ወክቡር፡
+                            እስከ፡ ለዓለመ፡ ዓለም፡ አሜን፡</incipit>
+                            <incipit xml:lang="gez" type="supplication"><sic>ቱ፡</sic> ወበረከቱ፡ ለቅዱስ፡ ሱስንዮስ፡ ያድኅና፡ እምሕማመ፡ ዓይነጥላ፡ ወዓይነናስ<supplied reason="omitted">፡</supplied>
+                                ዓይነናስ፡ ዓይነ፡ ዛር፡ ወዓይነ፡ ጠቢብ፡ ወቡዳ፡ ባርያ<supplied reason="omitted">፡</supplied> ወሌጌዎን፡ ድድቅ፡ ወጋኔነ፡ ቀትር፡ ምች፡ ወተላዋሽ፡
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:52]: ወቡዳ፡ ዓይነ፡ ባርያ

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 6]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                ቁረኛ፡ ወተያያዥ፡ ዕጀ፡ ሰብእ፡ ወጽላወጌ፡ ደም፡ ወሽዐተላይ<supplied reason="omitted">፡</supplied>
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:53]: ቁራኛ፡ ?
ወጽላወጊ፡ ?
ወሽተላይ

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 7]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                ለዓመትከ<hi rend="rubric" rendition="#partialRubric">፨</hi> 
+                                <persName ref="PRS13860HawituA" role="owner"><hi rend="rubric">ሐዊቱ አሊ፡</hi></persName>
+                                <persName ref="PRS13861WalattaH" role="owner"><hi rend="rubric"><add place="interlinear">ወለተ፡ ሕይወት</add></hi></persName></incipit>
+                            <incipit xml:lang="gez">ወሀሎ<supplied reason="omitted">፡</supplied> <hi rend="rubric" rendition="#partialRubric">፩</hi>
+                            ብእሲ፡ ዘስሙ፡ ሱስንዮስ፡ ወአውሰበ፡ ብእሲተ፡ ወወለደ፡ ወልደ፡ ትባዕተ፡ ወአንስተ፡ ወበቀዳማይ፡ ወልዳ፡ ቦዓት፡ ውርዝልያ፡ ጋኔን፡ 
+                                <del rend="expunctuated">ወ</del>ወቀተለቶ፡ ለወልዳ፡ ወበከየት፡ እሙ፡ ወይቤላ፡ ኦብእሲቶ፡ ምንት፡ ያበከየኪ፡</incipit>
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:59]: either ያበክየኪ፡ or, if the present form is kept, add sic?

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 8]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            <explicit xml:lang="gez">ወትቤ፡ ይእቲ፡ አንሰ፡ ኢየኃውር፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ውስቴታ፡ ወኢይቀርብ፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ወይፀውራ፡ 
+                                ለዛቲ፡ ጸሎት፡ እመሂ፡
+                            አንስት፡ ወእመሂ፡ <sic>አንስት፡</sic> ወወተባዕት፡ ወእመሂ፡ አዕሩግ፡ ወሕፃናት፡ ወወሬዛ፡ ልሂቅ፡ ኢይበጽሕ፡ እስከ፡ ለአለም፡ ወእምድኅረዝ፡ ሶበ፡ ኮነ፡ ሰማዕተ፡ ቅዱስ፡ ሱስንዮስ፡ በስመ፡
+                                እግዚእነ፡ ኢየሱስ<supplied reason="omitted">፡</supplied> <surplus>ኢየሱስ፡</surplus> ክርስቶስ፡ ሎቱ፡ ስብሐት፡ እስከ፡ ለዓለመ፡ አለም፡ አሜን።</explicit>
+                            <explicit xml:lang="gez" type="supplication">ኦአምላከ፡ ቅዱስ፡ ሱስንዮስ፡ አድኅና፡ እምሕማመ፡ ዓይነጥላ፡ ወዓይነናስ፡ ወዓይነናስ፡ ዓይነ፡ ዘር፡ ወውላጅ፡ ዓይነጠቢብ፡ ወቡዳ፡
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:64]: Omit one ወቡዳ፡

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 9]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                ዓይነናስ፡ ዓይነ፡ ዛር፡ ወዓይነ፡ ጠቢብ፡ ወቡዳ፡ ባርያ<supplied reason="omitted">፡</supplied> ወሌጌዎን፡ ድድቅ፡ ወጋኔነ፡ ቀትር፡ ምች፡ ወተላዋሽ፡
+                                ቁረኛ፡ ወተያያዥ፡ ዕጀ፡ ሰብእ፡ ወጽላወጌ፡ ደም፡ ወሽዐተላይ<supplied reason="omitted">፡</supplied>
+                                ለዓመትከ<hi rend="rubric" rendition="#partialRubric">፨</hi> 
+                                <persName ref="PRS13860HawituA" role="owner"><hi rend="rubric">ሐዊቱ አሊ፡</hi></persName>
+                                <persName ref="PRS13861WalattaH" role="owner"><hi rend="rubric"><add place="interlinear">ወለተ፡ ሕይወት</add></hi></persName></incipit>
+                            <incipit xml:lang="gez">ወሀሎ<supplied reason="omitted">፡</supplied> <hi rend="rubric" rendition="#partialRubric">፩</hi>
+                            ብእሲ፡ ዘስሙ፡ ሱስንዮስ፡ ወአውሰበ፡ ብእሲተ፡ ወወለደ፡ ወልደ፡ ትባዕተ፡ ወአንስተ፡ ወበቀዳማይ፡ ወልዳ፡ ቦዓት፡ ውርዝልያ፡ ጋኔን፡ 
+                                <del rend="expunctuated">ወ</del>ወቀተለቶ፡ ለወልዳ፡ ወበከየት፡ እሙ፡ ወይቤላ፡ ኦብእሲቶ፡ ምንት፡ ያበከየኪ፡</incipit>
+                            <explicit xml:lang="gez">ወትቤ፡ ይእቲ፡ አንሰ፡ ኢየኃውር፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ውስቴታ፡ ወኢይቀርብ፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ወይፀውራ፡
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:60]: ወኀበ፡ ይፀውራ፡

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 10]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            <incipit xml:lang="gez">ወሀሎ<supplied reason="omitted">፡</supplied> <hi rend="rubric" rendition="#partialRubric">፩</hi>
+                            ብእሲ፡ ዘስሙ፡ ሱስንዮስ፡ ወአውሰበ፡ ብእሲተ፡ ወወለደ፡ ወልደ፡ ትባዕተ፡ ወአንስተ፡ ወበቀዳማይ፡ ወልዳ፡ ቦዓት፡ ውርዝልያ፡ ጋኔን፡ 
+                                <del rend="expunctuated">ወ</del>ወቀተለቶ፡ ለወልዳ፡ ወበከየት፡ እሙ፡ ወይቤላ፡ ኦብእሲቶ፡ ምንት፡ ያበከየኪ፡</incipit>
+                            <explicit xml:lang="gez">ወትቤ፡ ይእቲ፡ አንሰ፡ ኢየኃውር፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ውስቴታ፡ ወኢይቀርብ፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ወይፀውራ፡ 
+                                ለዛቲ፡ ጸሎት፡ እመሂ፡
+                            አንስት፡ ወእመሂ፡ <sic>አንስት፡</sic> ወወተባዕት፡ ወእመሂ፡ አዕሩግ፡ ወሕፃናት፡ ወወሬዛ፡ ልሂቅ፡ ኢይበጽሕ፡ እስከ፡ ለአለም፡ ወእምድኅረዝ፡ ሶበ፡ ኮነ፡ ሰማዕተ፡ ቅዱስ፡ ሱስንዮስ፡ በስመ፡
+                                እግዚእነ፡ ኢየሱስ<supplied reason="omitted">፡</supplied> <surplus>ኢየሱስ፡</surplus> ክርስቶስ፡ ሎቱ፡ ስብሐት፡ እስከ፡ ለዓለመ፡ አለም፡ አሜን።</explicit>
+                            <explicit xml:lang="gez" type="supplication">ኦአምላከ፡ ቅዱስ፡ ሱስንዮስ፡ አድኅና፡ እምሕማመ፡ ዓይነጥላ፡ ወዓይነናስ፡ ወዓይነናስ፡ ዓይነ፡ ዘር፡ ወውላጅ፡ ዓይነጠቢብ፡ ወቡዳ፡
+                            ወቡዳ፡ ዓይነ፡ ባርያ<surplus>ያ</surplus>፡ ወሌጌዎን፡ ድድቅ፡ ወጋኔነ፡ ቀትር፡ ምች፡ ወተላኖሽ፡ ቁረኛ፡ ወተያይዦ፡ ደ፡
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:65]: ወተላዋሽ፡
ቁራኛ፡ ?
ወተያያዥ፡
ደም

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 11]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            ብእሲ፡ ዘስሙ፡ ሱስንዮስ፡ ወአውሰበ፡ ብእሲተ፡ ወወለደ፡ ወልደ፡ ትባዕተ፡ ወአንስተ፡ ወበቀዳማይ፡ ወልዳ፡ ቦዓት፡ ውርዝልያ፡ ጋኔን፡ 
+                                <del rend="expunctuated">ወ</del>ወቀተለቶ፡ ለወልዳ፡ ወበከየት፡ እሙ፡ ወይቤላ፡ ኦብእሲቶ፡ ምንት፡ ያበከየኪ፡</incipit>
+                            <explicit xml:lang="gez">ወትቤ፡ ይእቲ፡ አንሰ፡ ኢየኃውር፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ውስቴታ፡ ወኢይቀርብ፡ ኀበ፡ ሀሎ፡ ስምከ፡ ፍኖተ፡ ወይፀውራ፡ 
+                                ለዛቲ፡ ጸሎት፡ እመሂ፡
+                            አንስት፡ ወእመሂ፡ <sic>አንስት፡</sic> ወወተባዕት፡ ወእመሂ፡ አዕሩግ፡ ወሕፃናት፡ ወወሬዛ፡ ልሂቅ፡ ኢይበጽሕ፡ እስከ፡ ለአለም፡ ወእምድኅረዝ፡ ሶበ፡ ኮነ፡ ሰማዕተ፡ ቅዱስ፡ ሱስንዮስ፡ በስመ፡
+                                እግዚእነ፡ ኢየሱስ<supplied reason="omitted">፡</supplied> <surplus>ኢየሱስ፡</surplus> ክርስቶስ፡ ሎቱ፡ ስብሐት፡ እስከ፡ ለዓለመ፡ አለም፡ አሜን።</explicit>
+                            <explicit xml:lang="gez" type="supplication">ኦአምላከ፡ ቅዱስ፡ ሱስንዮስ፡ አድኅና፡ እምሕማመ፡ ዓይነጥላ፡ ወዓይነናስ፡ ወዓይነናስ፡ ዓይነ፡ ዘር፡ ወውላጅ፡ ዓይነጠቢብ፡ ወቡዳ፡
+                            ወቡዳ፡ ዓይነ፡ ባርያ<surplus>ያ</surplus>፡ ወሌጌዎን፡ ድድቅ፡ ወጋኔነ፡ ቀትር፡ ምች፡ ወተላኖሽ፡ ቁረኛ፡ ወተያይዦ፡ ደ፡ 
+                            ወሸተላይ፡ መገኛ፡ ወጕሥምት፡ ውጋት፡ ወቁርጥማት፡ ፍልሸት፡ ሥራይ፡ ወድግምት፡ ወቁርጸት፡ ፌራ፡ ዘደጋ፡ ንዳድ፡ ዘቆላ፡ ወእምኵሉ፡ ደዌያት፡
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:66]: ፍልጸት፡
ገዳድ፡ ?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]
Sorry for the minor suggestions – I will make use of the editing functionality for the rest of the record (when suitable).

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 12]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="1r122" to="1r219"/>
+                            <title type="complete" ref="LIT4703MagicPr"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ጸሎተ፡ ነዲራ፡ ዝውእቱ፡ ሕ</hi>ማመ፡
+                                ዓይነት፡ እኪት፡ ወእንዘ፡ የኃውር፡ እግዚእነ፡ ውስተ<supplied reason="omitted">፡</supplied> ባሕረ፡ ጥብርያዶስ፡
+                                ወምስ<hi rend="rubric">ሌሁ፡ ፲ወ፪ አርዳኢሁ፡
+                            ወሶበ፡ ርዕዩ፡ መልክዓ፡ ብእሲት፡ ዓራጊት፡ እንተ፡ ትነብ</hi>ር፡ በማዕዶተ፡ ፈለግ፡</incipit>
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:78]: ኀራጊት፡

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 13]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            <explicit xml:lang="gez">አድኅና፡ እምሕማመ፡ ደም፡ ወሾተላይ፡ ለዓመተ፡ እግዚአብሔር<supplied reason="omitted">፡</supplied>
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:79]: No word divider after እምሕማመ

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 14]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ጸሎተ፡ ነዲራ፡ ዝውእቱ፡ ሕ</hi>ማመ፡
+                                ዓይነት፡ እኪት፡ ወእንዘ፡ የኃውር፡ እግዚእነ፡ ውስተ<supplied reason="omitted">፡</supplied> ባሕረ፡ ጥብርያዶስ፡
+                                ወምስ<hi rend="rubric">ሌሁ፡ ፲ወ፪ አርዳኢሁ፡
+                            ወሶበ፡ ርዕዩ፡ መልክዓ፡ ብእሲት፡ ዓራጊት፡ እንተ፡ ትነብ</hi>ር፡ በማዕዶተ፡ ፈለግ፡</incipit>
+                            <explicit xml:lang="gez">አድኅና፡ እምሕማመ፡ ደም፡ ወሾተላይ፡ ለዓመተ፡ እግዚአብሔር<supplied reason="omitted">፡</supplied> 
+                                <persName ref="PRS13860HawituA"><hi rend="rubric">ሐዊቱ አሊ</hi></persName>
+                                <persName ref="PRS13861WalattaH"><hi rend="rubric"><add place="interlinear">ወለተ፡ ሕይወት፡</add></hi></persName>
+                                ሊስ፡ አፍሊስ<supplied reason="omitted">፡</supplied> መሊስ፡ ማላሊስ፡ መላልዮስ፡ በኃይለ፡ ዝንቱ፡ አስማቲከ፡ ዘአርጋዕከ፡ ኀይለ፡ በረድ፡ ወነፋስ፡ ከማሁ፡ አርግዕ፡ ደማ፡ 
+                                ወአፅገዕ፡ ፍሬ፡ ማኅፀና፡
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:83]: ወአፅግዕ፡ ?

File: OxfordBodleian/Juel-Jensen/BDLaethg40.xml [hunk 15]
```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                <persName ref="PRS13861WalattaH" role="owner"><hi rend="rubric"><add place="interlinear">ወለተ፡ ሕይወት</add></hi></persName></explicit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus from="1r219" to="1r230"/>
+                            <title type="complete" ref="LIT6739PPDam"/>
+                            <textLang mainLang="gez"/>
+                            <note>The following is the entire text.</note>
+                            <incipit xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ሕማመ፡ ደም፡ <gap reason="illegible"/></hi>
+                                ተላይ፡ ማሎክ፡ ሉሎክ፡ አፍላሎክ፡ በእለ፡ አስማቲከ፡ ዘአንሳእከ፡ ለዓልአዛር፡ እም<gap reason="illegible"/>ሰ፡ መቃብር፡ ከማሁ፡
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethg40.xml:95]: በእሉ፡ ?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
Thank you Jonas, I carried out the corrections

**Pull request merged**